common/static/common/templates/discussion/search-alert.underscore

```diff
@@ -4,6 +4,7 @@
     </div>
 
     <div class="search-alert-controls">
-      <a href="#" class="dismiss control control-dismiss"><i class="icon fa fa-remove"></i></a>
+      <a href="#" class="dismiss control control-dismiss"><span class="icon fa fa-remove" aria-hidden="true"></span>
+      <span class="sr"><%= gettext("Close") %></span></a>
```

Comment:
<%**-** gettext("Close") %> instead of <%**=** gettext("Close") %> ? What's the difference?